content/releases/Release Notes 11.20.25.md

```
@@ -0,0 +1,29 @@
+---
+title: Release Notes 11.20.25
+date: 11.20.25
+tags:
+  - releases
+  - announcements
+  - honcho
+  - dev
+---
+# HONCHO v2.4.3
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)